ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.tsx

```diff
@@ -0,0 +1,252 @@
+import React from 'react';
+import {
+    Flex,
+    FlexItem,
+    Stack,
+    StackItem,
+    Text,
+    TextContent,
+    TextVariants,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+} from '@patternfly/react-core';
+import {
+    ExpandableRowContent,
+    TableComposable,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+} from '@patternfly/react-table';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+import './DeploymentFlows.css';
+
+interface FlowBase {
+    type: 'Deployment' | 'External';
+    entity: string;
+    namespace: string;
+    direction: string;
+    port: string;
+    protocol: string;
+    isAnomalous: boolean;
+}
+
+interface Flow extends FlowBase {
+    children: FlowBase[];
+}
```

Comment:
i wonder if it would be useful to have these in a separate `types.ts` file in this directory for easy access

Reply:
I wouldn't expect this type to be used between many components, but it might be useful when we get to the baselines tab. For now, I'll keep it here, and if required we can put it in its own type file.